provider.tf

```diff
@@ -0,0 +1,18 @@
+provider "aws" {
+  region = var.region
```

Comment:
no provider config in module, have provider requirement for primary and secondary (virginia) region providers
please have some minimum outputs
rename providr.tf to versions.tf